compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/window/SwingDialog.desktop.kt

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright 2025 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package androidx.compose.ui.window
```

Comment:
I would move it to `androidx.compose.ui.awt` to be consistent with `SwingPanel` Composable and `ComposeWindow` class.

On the other hand, it will be inconsistent with `AwtWindow`, so we would have to move it too.

What do you think is better?

Reply:
Moved, and also improved some of the documentation on related functions.